ghdocs/TUTORIAL.md

```diff
@@ -323,24 +507,235 @@ To accomplish this, let's add the following new styles to app.css:
 
-Now let's tweak our list a bit. For our purposes, we really want to just have a title and description with the x and checkmark. Since we don't need the paper clip and flag, let's modify one list item to look like the following:
+**Hang on, this part gets a bit tricky**
+
+We really want to tweak our list component for our purposes, we want to have a title and description with the x and checkmark. Since we don't need the paper clip and flag, go ahead and remove all `<div class="ms-ListItem is-selectable"></div>` elements **Except the first one** this will give us one element to work with. 
+
+Inside of the first `.ms-ListItem` remove the elements` <div class="ms-ListItem-action"><i class="ms-Icon ms-Icon--mail"></i></div>` and `<div class="ms-ListItem-action"><i class="ms-Icon ms-Icon--trash"></i></div>`. Then changed `ms-Icon ms-Icon--flag` to `ms-Icon ms-Icon--x` and `ms-Icon ms-Icon--pinLeft` to `ms-Icon ms-Icon--check`. our new list html should look like the following: 
 
 ```html
-<li class="ms-List-item">
-  <div class="ms-ListItem is-unread">
-    <span class="ms-ListItem-primaryText">Alton Lafferty</span>
-    <span class="ms-ListItem-tertiaryText">Today we discussed the importance of a, b, and c in regards to d.</span>
-    <span class="ms-ListItem-metaText">2:42p</span>
-    <div class="ms-ListItem-selectionTarget js-toggleSelection"></div>
-    <div class="ms-ListItem-actions">
-      <div class="ms-ListItem-action"><i class="ms-Icon ms-Icon--x"></i></div>
-      <div class="ms-ListItem-action"><i class="ms-Icon ms-Icon--check"></i></div>
+   <div class="TodoList">
+        <ul class="ms-List">
+            <div class="ms-ListItem is-unread is-selectable">
+                <span class="ms-ListItem-primaryText">Alton Lafferty</span>
+                <span class="ms-ListItem-secondaryText">Meeting notes</span>
+                <span class="ms-ListItem-tertiaryText">Today we discussed the importance of a, b, and c in regards to d.</span>
+                <span class="ms-ListItem-metaText">2:42p</span>
+                <div class="ms-ListItem-selectionTarget js-toggleSelection"></div>
+                <div class="ms-ListItem-actions">
+                    <div class="ms-ListItem-action"><i class="ms-Icon ms-Icon--x"></i></div>
+                    <div class="ms-ListItem-action"><i class="ms-Icon ms-Icon--check"></i></div>
+                </div>
+            </div>
+        </ul>
     </div>
-  </div>
-</li>
 ```
 
-Now let's copy this new list item code and paste it at least 10 times inside of `<ul class=”ms-List”></ul>`, making sure to remove the old list item code.
+Now let's copy this new list item code and paste it at least 10 times inside of `<ul class=”ms-List”></ul>`. Afterwards  you should now be complete with the first part of this tutorial below is the complete HTML and CSS.
```

Comment:
I'd rephrase to:
"Now let's copy this new list item code and paste it at least 10 times inside of `<ul class=”ms-List”></ul>` to fill out the UI of the app. With this final step, you just completed the tutorial! Below is the complete HTML and CSS:"